eMPRess_GUI_version1.py

```diff
@@ -0,0 +1,354 @@
+
+        ### creates an input frame on the left side of the master frame ###
+        self.func_frame = tk.Frame(master)
+        self.func_frame.grid(row=1, column=0, sticky="nsew")
+        self.func_frame.grid_rowconfigure(0, weight=1)
+        self.func_frame.grid_rowconfigure(1, weight=1)
+        self.func_frame.grid_rowconfigure(2, weight=1)
+        self.func_frame.grid_rowconfigure(3, weight=1)
+        self.func_frame.grid_columnconfigure(0, weight=1)
+        self.func_frame.grid_propagate(False)
+
+        ### creates an output frame on the right side of the master frame ###
+        self.output_frame = tk.Frame(master)
+        self.output_frame.grid(row=1, column=1, sticky="nsew")
+        self.output_frame.grid_rowconfigure(0, weight=1)
+        self.output_frame.grid_rowconfigure(1, weight=1)
+        self.output_frame.grid_rowconfigure(2, weight=1)
+        self.output_frame.grid_columnconfigure(0, weight=1)
+        self.output_frame.grid_propagate(False)
+
+        ### "Load File" button ###
+        # loads in an input .newick file
+        # and displays the number of leaves in each tree (DEMO for now) and entry boxes for DTL costs
+        load_file_btn = tk.Button(self.func_frame, text="Load File", background="green", height=2, width=9, command=self.load_file)
+        load_file_btn.grid(row=0, column=0)
+        # creates a Label to overwrite the old file path 
+        self.file_path_label = tk.Label(self.output_frame)
+
+        ### "View Event Cost Landscape" button ###
+        # pops up a matplotlib graph for the cost landscape
+        self.view_cost_btn = tk.Button(self.func_frame, text="View Event Cost Landscape", command=self.plot_cost_landscape, state=tk.DISABLED)
+        self.view_cost_btn.grid(row=1, column=0)
+
+        ### "Compute Reconciliations" button ###
+        # displays reconciliation results(numbers) and three options(checkboxes) for viewing graphical analysis
+        self.compute_recon_button = tk.Button(self.func_frame, text="Compute Reconciliations", command=self.recon_analysis, state=tk.DISABLED)
+        self.compute_recon_button.grid(row=2, column=0)
+
+    def load_file(self):
+        """Loads in an input file and displays the number of leaves in each tree when "Load File" button is clicked."""
+        # creates a frame to display the number of leaves and the file path
+        recon_input_info_frame = tk.Frame(self.output_frame)
+        recon_input_info_frame.grid(row=0, column=0, sticky="nsew")
+        recon_input_info_frame.grid_columnconfigure(0, weight=1)
+        recon_input_info_frame.grid_rowconfigure(0, weight=1)
+        recon_input_info_frame.grid_rowconfigure(1, weight=1)
+        recon_input_info_frame.grid_rowconfigure(2, weight=1)
+        recon_input_info_frame.grid_propagate(False)
+
+        # allows loading a .newick file
+        self.file_path = ""
+        input_file = tk.filedialog.askopenfilename(initialdir="/desktop", title="Select a file")
```

Comment:
Is there a way to select `initialdir` to the current directory that the user is running the script from? I think that might be a better default than "desktop". If you could think of a better default, feel free to use it. Desktop is okay too I think. On a side note, does using `/desktop` as the initial directory works? For example, my desktop is located at `~/desktop` which is short for `/Users/santisantichaivekin/desktop` but not at `/desktop`.

Reply:
It didn't work I think. I used "import os" and "os.getcwd()" to set to the current working directory. I feel that we can also set it to desktop in this way.